platforms/vitis/cl_firesim/Makefile

```diff
@@ -35,13 +35,22 @@ run-sim:   VPP_TARGET_SPECIFIC_ARGS = --target $(TARGET) --config $(buildroot)/s
 # current support on firesim1
 DEVICE ?= xilinx_u250_gen3x16_xdma_3_1_202020_1
 
+# Selects a vitis config, by looking in build-strategies, for an equivalently
+# named cfg file.
+STRATEGY ?= TIMING
+
+# If set, this will very first config file vitis sees.
+CONFIG_OVERLAY ?=
```

Comment:
Why is this needed?

Reply:
It's useful for people like me that want to experiment on the configs. It doesn't make sense to me to shoe horn all of this behind the build strategies flag + this lets me point at files that might be elsewhere on the system.

Reply:
Unless you mean why is this declaration needed? In that case, i think it just makes it more clear what the IF is.

Reply:
The 1st response was sufficient. Thanks